.github/workflows/docker.yml

```diff
@@ -0,0 +1,39 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
Just one note, this repo does not have a `master` branch, but `main`

Reply:
Resolved in [bc4ea37](https://github.com/gnolang/faucet/pull/7/commits/bc4ea37dfb14533c1d93f1f8de2c1a9460c2300d)